### DISCUSSION TITLE
Patching Gaps in the CIS Windows 11 Benchmark - BitLocker - odds+endpoints

### DISCUSSION THREAD

**Discussion by giscus** [ennnbeee/oddsandendpoints-web#2] [Comments]
# Patching Gaps in the CIS Windows 11 Benchmark - BitLocker

Updating the Center for Internet Security (CIS) benchmark build kit for Windows 11 in Microsoft Intune for BitLocker encryption and DMA protection.

https://memv.ennbee.uk/posts/windows-cis-patching-gaps-part1/

**Comment by jydareau** [comment]
Windows 11 build kit for Microsoft Intune unable to find in the CIS , Only PDF files

**Reply by ennnbeee** [reply]
The 3.0.1 build kit for Windows 11 is available [here](https://workbench.cisecurity.org/files/5179), once you've registered and signed into the CIS workbench site.

**Comment by jydareau** [comment]
I am Log in the CIS   404 Page Not Found
________________________________
De : Nick Benton ***@***.***>
Envoyé : mercredi 19 juin 2024 08:17
À : ennnbeee/ennnbeee.github.io ***@***.***>
Cc : jydareau ***@***.***>; Comment ***@***.***>
Objet : Re: [ennnbeee/ennnbeee.github.io] Patching Gaps in the CIS Windows 11 Benchmark - BitLocker · MEM v ENNBEE - Battles with Modern Device Management (Discussion #2)


The 3.0.1 build kit for Windows 11 is available here<https://workbench.cisecurity.org/files/5179>, once you've registered and signed into the CIS workbench site.

**Reply by ennnbeee** [reply]
You could try searching for the Window 11 Intune Build Kit, as it is available when I'm logged in:

<img width="600" alt="image" src="https://github.com/ennnbeee/ennnbeee.github.io/assets/47244836/c776f68e-5f75-45a8-9dd8-7f7882acf6b2">

I'll upload the amended CIS files to my github repo shortly.

**Comment by jydareau** [comment]
CIS Microsoft Intune for Windows 11 Benchmark v3.0.1
 I find [cid:11fc1509-666e-47ed-b057-2cfcfbdb5ddc]
CIS Microsoft Intune for Windows 11 Benchmark v3.0.1 - Build Kit
I cant find build kit, may is the level of my account



________________________________
De : Nick Benton ***@***.***>
Envoyé : mercredi 19 juin 2024 08:25
À : ennnbeee/ennnbeee.github.io ***@***.***>
Cc : jydareau ***@***.***>; Comment ***@***.***>
Objet : Re: [ennnbeee/ennnbeee.github.io] Patching Gaps in the CIS Windows 11 Benchmark - BitLocker · MEM v ENNBEE - Battles with Modern Device Management (Discussion #2)


You could try searching for the Window 11 Intune Build Kit, as it is available when I'm logged in:

image.png (view on web)<https://github.com/ennnbeee/ennnbeee.github.io/assets/47244836/c776f68e-5f75-45a8-9dd8-7f7882acf6b2>

I'll upload the amended CIS files to my github repo shortly.

**Comment by jjarose** [comment]
Thank you for this. I found your "how to" late last week and read it this morning. It just so happens, in today's CIS Intune Community meeting, (open to the public) is focusing on BitLocker settings in our Intune Benchmark....So good changes are coming with the next release. :)  I am going to share this with those that show up.   

Again, thank you and feel free to join one of our meetings, we are always looking for SMEs to help shape our benchmarks. 

Jen
Sr. Cybersecurity Engineer, CIS

**Reply by ennnbeee** [reply]
Thanks for reading through the series, I hope you didn't mind the slightly frank and blunt method of the review 😅.

I've joined the CIS Intune community so will keep an eye on the discssions, and chip in where I can.

**Comment by jjarose** [comment]
Hi Nick,

I don’t mind at all. Without posts like this, and SMEs that give feedback, join the community, and community calls, we would not be able to do what we do.

Thanks again!
Jen


Jen Jarose
Senior Cybersecurity Engineer
***@***.******@***.***>
USA Eastern Time (ET)

[A close-up of a logo  Description automatically generated]<https://www.cisecurity.org/>

[A blue circle with white letters  Description automatically generated]<https://www.linkedin.com/company/the-center-for-internet-security/>  [A blue circle with white x in it  Description automatically generated] <https://twitter.com/CISecurity>   [A blue circle with a letter f in it  Description automatically generated] <https://www.facebook.com/CenterforIntSec>   [A blue circle with a play button  Description automatically generated] <https://www.youtube.com/user/TheCISecurity>   [A blue circle with a white square with a circle in the middle  Description automatically generated] <https://www.instagram.com/cisecurity>

Center for Internet Security
31 Tech Valley Drive
East Greenbush, NY 12061

From: Nick Benton ***@***.***>
Sent: Thursday, August 22, 2024 9:41 AM
To: ennnbeee/ennnbeee.github.io ***@***.***>
Cc: Jennifer Jarose ***@***.***>; Comment ***@***.***>
Subject: [External] Re: [ennnbeee/ennnbeee.github.io] Patching Gaps in the CIS Windows 11 Benchmark - BitLocker · MEM v ENNBEE - Battles with Modern Device Management (Discussion #2)




Thanks for reading through the series, I hope you didn't mind the slightly frank and blunt method of the review 😅.

I've joined the CIS Intune community so will keep an eye on the discssions, and chip in where I can.

—
Reply to this email directly, view it on GitHub<https://github.com/ennnbeee/ennnbeee.github.io/discussions/2#discussioncomment-10419622>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUQKLXML5Y6ZLIBXRUBXDSLZSXS7VAVCNFSM6AAAAABJRKYU7KVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANBRHE3DEMQ>.
You are receiving this because you commented.Message ID: ***@***.***>

.....
This message and attachments may contain confidential information. If it appears that this message was sent to you by mistake, any retention, dissemination, distribution or copying of this message and attachments is strictly prohibited. Please notify the sender immediately and permanently delete the message and any attachments.

. . . . .

**Comment by Fr3d** [comment]
In our experience, on Win11 24H2, setting both "Configure TPM startup" and "Configure TPM startup PIN" to Allow sometimes resulted in BitLocker prompting for a password every boot which could then be bypassed by pressing Esc, as it would fall back to the regular Tpm unlock.

Adding the below lines to your very useful `CIS (BL) BitLocker TPMandPIN - Windows 11 Intune 3.0.1.ps1` script (around line 10) ensures the regular Tpm key protector gets removed as well:
```powershell
  # Detects and removes existing Tpm key protectors to ensure the PIN is required
  if ($osVolume.KeyProtector.KeyProtectorType -contains 'Tpm') {
    $osVolume.KeyProtector | Where-Object { $_.KeyProtectorType -eq 'Tpm' } | ForEach-Object {
      Remove-BitLockerKeyProtector -MountPoint $osVolume.MountPoint -KeyProtectorId $_.KeyProtectorId
    }
  }
```

Finally, thanks for the fantastic series of posts!

**Reply by ennnbeee** [reply]
Thanks for this, I've updated the PowerShell script and the post with the information provided.
